src/mlpack/methods/ann/layer/max_pooling_impl.hpp

```diff
@@ -31,24 +31,82 @@ MaxPooling<InputDataType, OutputDataType>::MaxPooling(
     const size_t kernelHeight,
     const size_t strideWidth,
     const size_t strideHeight,
-    const bool floor) :
+    const bool floor,
+    const size_t inputWidth,
+    const size_t inputHeight,
+    const size_t padW,
+    const size_t padH,
+    const std::string paddingType) :
+    MaxPooling(
+    kernelWidth,
+    kernelHeight,
+    strideWidth,
+    strideHeight,
+    floor,
+    inputWidth,
+    inputHeight,
+    std::tuple<size_t, size_t>{padW, padW},
```

Comment:
This is something you can completely ignore. Generally around the code base we initialized tuples like this 

`std::tuple<size_t, size_t>(padW, padW)`

I know it doesn't make any difference, if you want you can change it or leave it as it is. Thanks.

Reply:
I changed it for the sake of consistency.